static/app/views/performance/newTraceDetails/traceHeader.tsx

```diff
@@ -20,27 +20,31 @@ import type {UseApiQueryResult} from 'sentry/utils/queryClient';
 import type RequestError from 'sentry/utils/requestError/requestError';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
-import {getTraceInfo} from '../traceDetails/utils';
 import {BrowserDisplay} from '../transactionDetails/eventMetas';
 import {MetaData} from '../transactionDetails/styles';
 
+import type {TraceTree} from './traceTree';
+
 type TraceHeaderProps = {
   metaResults: UseApiQueryResult<TraceMeta | null, any>;
   organization: Organization;
   rootEventResults: UseApiQueryResult<EventTransaction, RequestError>;
   traces: TraceSplitResults<TraceFullDetailed> | null;
+  tree: TraceTree;
 };
 
 export default function TraceHeader({
   metaResults,
   rootEventResults,
   traces,
   organization,
+  tree,
 }: TraceHeaderProps) {
-  const errors = metaResults.data?.errors || 0;
-  const performanceIssues = metaResults.data?.performance_issues || 0;
+  const traceNode = tree.root.children[0];
```

Comment:
This can be undefined, you need to guard properly or L44 and L45 will throw  an error